scripts/README.md

```diff
@@ -0,0 +1,173 @@
+# Script Usage
+
+This directory contains a mix of a few different types of scripts that are useful for interacting with the Gnosis Protocol in several different envronments (i.e. Mainnet, Rinkeby & Development) and for a variety of different purposes such as
+
+1. **Elementary Interactions** (e.g. token listing, deposit, request and claim withdraw, order placement and cancellation)
+2. **Data Querying** (invoke view functions, get auction elements, transitive orderbook)
+3. **Involved Interactions** (add token list, ensure owl liquidity, place spread orders)
+4. **Development Interactions** (wait seconds, close auction, setup environment, setup thegraph data)
```

Comment:
I'm slightly worried that listing all scripts out here specifically might become outdated rather quickly.

Reply:
Well, it doesn't necessarily have to be a list of all scripts. Notice that no claim about all of them is explicitly stated and these brackets are meant to indicate that they are just a subset (although they do happen to be all at the current moment). 

I don't think having nothing here is the best solution, but I could remove the last two examples along with the sentence "In the essence of completeness..."